jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/ManifestPullerIntegrationTest.java

```diff
@@ -80,31 +112,41 @@ public void testPull_v22ManifestList() throws IOException, RegistryException {
         RegistryClient.factory(EventHandlers.NONE, "gcr.io", "distroless/base", httpClient)
             .newRegistryClient();
 
-    // Ensure ":latest" is a manifest list
+    // Ensures call to image at KNOWN_MANIFEST_LIST_SHA returns a manifest list
```

Comment:
minor: how about we say something like "at the specified sha`? This can prevent the comment from becoming stale if the variable name changes.

Reply:
Makes sense, updated!